openapi/endpoints/prerequisites/models/get-spending-prerequisite-response.yaml

```diff
@@ -9,7 +9,10 @@ properties:
           type: string
           enum:
             - smart_contract_write
+            - kyc
         params:
           oneOf:
```

Comment:
I noticed when you have the example from schema section it just shows the first model. I think this is ok - what do you guys think? 

This is the example from schema:
```
{
  "prerequisites": [
    {
      "type": "smart_contract_write",
      "params": {
        "contractAddress": "string",
        "abi": {
          "type": "function",
          "name": "approve",
          "inputs": [
            {
              "name": "_spender",
              "type": "address"
            }
          ],
          "outputs": [
            {
              "name": "string",
              "type": "uint256"
            }
          ]
        },
        "method": "string",
        "params": {}
      }
    }
  ]
}
```